src/h5web/vis-packs/nexus/utils.ts

```diff
@@ -20,35 +21,29 @@ import {
   assertStringType,
   assertArray,
   isDefined,
+  isGroup,
 } from '../../guards';
 import type { NxData, SilxStyle } from './models';
 import { isScaleType } from '../core/utils';
 import { getAttributeValue, getChildEntity } from '../../utils';
 
-export function getDatasetLabel(dataset: Dataset): string {
-  const longName = getAttributeValue(dataset, 'long_name');
-  if (longName && typeof longName === 'string') {
-    return longName;
-  }
-
-  const units = getAttributeValue(dataset, 'units');
-  if (units && typeof units === 'string') {
-    return `${dataset.name} (${units})`;
-  }
-
-  return dataset.name;
-}
```

Comment:
Just moving this utility down to the bottom, since it is chronologically called after all the others (not that it's a strict rule, it just feels more logical to have it after `getNxData` in this case).